MiniProject1.py

```diff
@@ -0,0 +1,96 @@
+# Kimber's file
+from pattern.en import *
+rhymeWords=dict() #dictionary of words that rhyme
+generalWords=dict() #markov chain dictionary
+#fo = open("ShakespeareSonnets.txt", "r")
+with open('ShakespeareSonnets.txt', 'r') as f:
+	read_data = f.read()
+f.closed
+sonnetz= read_data.split('\n\n')
+sonnetLine=[0, 1, 2, 3,4, 5,6 ,7, 8, 9, 10, 11, 12, 13]
+import random
```

Comment:
A little thing, but try to keep all your import statements at the top. That way, someone just casually reading through your code will know what libraries this code needs right away.